tests/iDibo_tests.py

```diff
@@ -1,11 +1,35 @@
-from nose.tools import *
-import iDibo
+import unittest
+from gluon.shell import exec_environment
+from gluon.contrib.test_helpers import form_postvars
```

Comment:
Is there a reason why you need that module under `web2py/gluon`? 
If you just need a helper for the tests, you can add it in the same `iDibo/tests` directory (the root of the github repository) and import it simply by `from test_helpers import form postvars`

If there is a good reason why you would need it under the `web2py/gluon` dependency folder, you can, for the moment, commit it under a new folder called `iDibo/ext` and I will move it by hand to the right location. This is not the right way to handle dependency management, we'll clean up in issue #13 

I'm now starting to review everything else